edx-toggles/.gitignore

```diff
@@ -0,0 +1,79 @@
+*.py[cod]
```

Comment:
Where did this .gitignore come from?  The Django cookiecutter (https://github.com/edx/cookiecutter-django-app/tree/master/%7B%7Bcookiecutter.repo_name%7D%7D/%7B%7Bcookiecutter.app_name%7D%7D) doesn't have it.  It seems redundant with the one in the root.

Reply:
@nedbat I wouldn't worry about reviewing this one yet. I was just trying out the cookiecutter and wanted to be able to read it on my phone. I plan on cleaning it up this week